db/postgres/search-bool.go

```diff
@@ -0,0 +1,46 @@
+package pgUtils
+
+import (
+	"fmt"
+
+	"github.com/Masterminds/squirrel"
+
+	"github.com/monstercat/golib/operator"
+)
+
+func NewBoolOperator(field string, keys ...string) SearchOperatorConfigBool {
+	return SearchOperatorConfigBool{
+		SearchOperatorConfigBase{
+			Field: field,
+			Keys: keys,
+		},
+	}
+}
+
+type SearchOperatorConfigBool struct {
+	SearchOperatorConfigBase
+}
+
+func parseBool(o string) bool {
+	if o == "" {
+		return false
+	}
+	switch o[0] {
+	case 't', 'T', '1':
+		return true
+	}
```

Comment:
Should handle "true"? Can also use strings.ToLower here as well. I would only perform 1 byte check if length is 1 byte. Otherwise 'too' passes and I'm not sure that's a good thing...